staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -215,6 +215,9 @@ type Cacher struct {
 	// watchersBuffer is a list of watchers potentially interested in currently
 	// dispatched event.
 	watchersBuffer []*cacheWatcher
+	// watchersSlower is a list of watchers whose buffer are full in currently
```

Comment:
.. whose buffer is currently full

(i.e. s/in currently dispatched event.//)

Reply:
done